Makefile

```diff
@@ -4,10 +4,6 @@ BUILD_DATE=`date -R`
 VERSION=$(shell git describe --tags)
 GOVERSION=$(shell go version)
 
-ENV=updateCli.env
-
-include $(ENV)
```

Comment:
As mentioned in https://github.com/olblak/updatecli/pull/117#issuecomment-724665899 this part is not needed.